src/js/components/tasks/cards/confirmRemoveUserModal.tsx

```diff
@@ -0,0 +1,66 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Modal from '@salesforce/design-system-react/components/modal';
+import i18n from 'i18next';
+import React from 'react';
+
+import { AssignedUserTracker } from '@/components/tasks/cards';
+import { Org, OrgsByTask } from '@/store/orgs/reducer';
+import { ORG_TYPES } from '@/utils/constants';
+
+const ConfirmRemoveUserModal = ({
+  orgs,
+  assignedUser,
+  handleClose,
+  handleDelete,
+}: {
+  orgs: OrgsByTask;
+  assignedUser: AssignedUserTracker | null;
+  handleClose: () => void;
+  handleDelete: (
+    org: Org,
+    shouldRemoveUser?: AssignedUserTracker | null,
+  ) => void;
+}) => {
+  const type = assignedUser?.type;
+  const handleSubmit = () => {
+    const org = type && orgs[type];
+    handleClose();
+    /* istanbul ignore else */
+    if (org && assignedUser) {
+      handleDelete(org, { ...assignedUser });
+    }
+  };
+  const heading =
+    type === ORG_TYPES.QA
+      ? i18n.t('Confirm Change Reviewer and Delete Review Org')
+      : i18n.t('Confirm Change Developer and Delete Dev Org');
```

Comment:
Could these titles say "Confirm Reviewer Change ..." or "Confirm Changing the Reviewer ..." it reads a little funny as its currently written IMO. Same for Developer